### PR TITLE
This is my silly branch

### PR DESCRIPTION
**Pull request by worldofgeese**
This branch is composed of the magnificent work of my peers.

### CODE REVIEW

File: hello-world.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+Greetings from Billund Campus
+
 I love muffins!
 
 I love chocolate
 
 But chocolate is unhealthy for you
+
+## What is your favorite type of dessert?
+
```
Comment (valemorra) [inline hello-world.md:12]: buuuuuuu

File: hello-world.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
+*Some* chocolate is unhealthy for you. Dark chocolate over 70% can be healthy for you.
+
+But some people might say the 80% kind is healthy
```
Comment (patLEGO) [inline hello-world.md:15]: Is this a scientific fact?

File: hello-world.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
+## Hello everyone
```
Comment (korolanna) [inline hello-world.md:16]: Hello! 👯

**Review by korolanna — APPROVED** [review]
Amazing collaboration here!

**Review by patLEGO — APPROVED** [review]
I approve this amazing git

File: hello-world.md (hunk 1 above)
Comment (worldofgeese) [reply hello-world.md:12]: Hey, thanks chief, I think I understood your request that all chocolate is healthy for you.

Comment (valemorra) [reply hello-world.md:12]: Acceptable

**Review by valemorra — APPROVED** [review]

**Pull request merged**